ui/react-components/components/EditAppointment/EditAppointment.jsx

```diff
@@ -254,11 +341,24 @@ const EditAppointment = props => {
         defaultTime: appointmentDetails.endTime
     };
 
+    const isStartDateModified = () => !isEqual(moment(originalAppointmentDate).format('DMYYYY'), moment(appointmentDetails.appointmentDate).format('DMYYYY'));
+
+    const isEndDateModified = () => !isEqual(moment(originalRecurringEndDate).format('DMYYYY'), moment(appointmentDetails.recurringEndDate).format('DMYYYY'));
+
```

Comment:
Two functions do the same work, parametrize and make it one function

Reply:
DONE